### PR TITLE
catalog#1

### PR DESCRIPTION
**Pull request by AndriiYelieva**
- [DEMO LINK](https://AndriiYelieva.github.io/layout_catalog/)
- [TEST REPORT LINK](https://AndriiYelieva.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/header.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+.nav__link {
+  padding: 23px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
```
Comment (OlEzhA0) [inline src/styles/header.scss:33]:
```suggestion
  font-weight: 500;
  font-size: 12px;
```

File: src/styles/nav.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
```
Comment (OlEzhA0) [inline src/styles/nav.scss:20]:
```suggestion
  padding: 50px 40px;

  &__card {
    margin-right: 48px;
    margin-bottom: 46px;

    &:nth-child(4n) {
      margin-right: 0;
    }

    &:nth-last-child(-n + 4) {
      margin-bottom: 0;
    }
  }
```

File: src/styles/nav.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.card {
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $white-color;
+  box-sizing: border-box;
```
Comment (OlEzhA0) [inline src/styles/nav.scss:29]: move it to `*` selector

File: src/styles/nav.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+  &_cards {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
```
Comment (OlEzhA0) [inline src/styles/nav.scss:34]: I don't see this class into your html

File: src/styles/nav.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    line-height: 14px;
+    padding-bottom: 16px;
+  }
+
+  &__review-container {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    align-items: flex-end;
+    text-align: right;
```
Comment (OlEzhA0) [inline src/styles/nav.scss:63]: it's redundant

File: src/styles/nav.scss [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    background-color: $main-color;
+  }
+
+  &__buy:hover {
+    color: $main-color;
+    background-color: $white-color;
+  }
+}
+
+.stars {
```
Comment (OlEzhA0) [inline src/styles/nav.scss:115]: move stars to separate file

File: src/styles/nav.scss [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background: url(../images/star-active.svg) no-repeat 50%;
+}
```
Comment (OlEzhA0) [inline src/styles/nav.scss:130]: add active stars for all `1, 2, 3, 4, 5` counts

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]

File: src/styles/header.scss [hunk 8]
```diff
@@ -0,0 +1,57 @@
+html {
+  margin: 0;
```
Comment (tetiana-ivanchenko) [inline src/styles/header.scss:2]: you shouldn't reset indentation for html -  it hasn't default margins

File: src/styles/nav.scss [hunk 9]
```diff
@@ -0,0 +1,109 @@
+@import "./stars.scss";
+
+body {
+  margin: 0;
+}
```
Comment (tetiana-ivanchenko) [inline src/styles/nav.scss:5]: move it to the main file

File: src/styles/nav.scss [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+.catalog {
```
Comment (tetiana-ivanchenko) [inline src/styles/nav.scss:7]: create separate files for the catalog and the card. why the file with this blocks call nav?

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix comments and also test report and the demo link

File: src/index.html [hunk 11]
```diff
@@ -1,12 +1,310 @@
...
+            <a href="#smartphones" class="nav__link"> 
+              Smartphones 
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#laptops&computers" 
+            class="nav__link" 
+            data-qa="nav-hover"
+            >
```
Comment (loralevitska) [inline src/index.html:50]: Fix all cases
```suggestion
            <a 
              href="#laptops&computers" 
              class="nav__link" 
              data-qa="nav-hover"
            >
```

File: src/styles/card.scss [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-accent-color;
+  }
+
+  &__code {
+    font-weight: 400;
```
Comment (loralevitska) [inline src/styles/card.scss:26]: Remove redundant default value

File: src/styles/catalog.scss [hunk 13]
```diff
@@ -0,0 +1,20 @@
+.catalog {
+  width: 1024px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```
Comment (loralevitska) [inline src/styles/catalog.scss:10]: Combine it with `margin` property

File: src/styles/header.scss [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__link {
+  padding: 23px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
```
Comment (loralevitska) [inline src/styles/header.scss:26]: It is a good practice to make all colors variable

File: src/styles/nav.scss (hunk 7 above)
Comment (loralevitska) [reply src/styles/nav.scss]: not fixed

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Fix previous and new comments

File: src/index.html [hunk 15]
```diff
@@ -1,12 +1,310 @@
...
+          </div>
+          <div class="card__price-container">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+          <a 
+          class="card__buy" 
+          href="#" 
+          data-qa="card-hover"
+          >
```
Comment (OlEzhA0) [inline src/index.html:107]:
```suggestion
          <a 
            class="card__buy" 
            href="#" 
            data-qa="card-hover"
          > 
```

File: src/styles/card.scss [hunk 16]
```diff
@@ -0,0 +1,87 @@
+@import "./stars.scss";
+@import "./catalog.scss";
+
+$cod-color: #616070;
```
Comment (OlEzhA0) [inline src/styles/card.scss:4]:
```suggestion
$code-color: #616070;
```

Comment (OlEzhA0) [reply src/styles/card.scss:4]: move it to your main file

File: src/styles/card.scss [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+    line-height: 18px;
+    font-weight: 700;
+    color: $text-accent-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
```
Comment (OlEzhA0) [inline src/styles/card.scss:70]: it's redundant

File: src/styles/catalog.scss (hunk 13 above)
Comment (OlEzhA0) [reply src/styles/catalog.scss:10]: not fixed
use
```suggestion
    margin: 0 48px 46px 0;
```

File: src/styles/header.scss [hunk 18]
```diff
@@ -0,0 +1,55 @@
+$color-link: #060b35;
```
Comment (OlEzhA0) [inline src/styles/header.scss:1]: move variables to your main or create separate file for vars

File: src/index.html [hunk 19]
```diff
@@ -1,12 +1,310 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+  >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
```
Comment (OlEzhA0) [inline src/index.html:25]: use BEM
```suggestion
      <a href="#" class="header__logo">
```

File: src/styles/main.scss [hunk 20]
```diff
@@ -1,3 +1,35 @@
-body {
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+$border-color: #f3f3f3;
+$white-color: #fff;
+
+* {
+  background: $white-color;
```
Comment (OlEzhA0) [inline src/styles/main.scss:8]: why do we need to define background here?

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]

**Review by OlEzhA0 — APPROVED** [review]

**Pull request closed without merging**